bin/sv_varcall_penalties.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+from pysam import VariantFile
+import cmdvcf
+import argparse
+from pprint import pprint
+from geneticmodels import GM
+
+MANTA_SIZE_FOR_PENALTY = 100000
+
+def main(args: object):
```

Comment:
`argparse.Namespace`

or, my preference:

```
def main(input_vcf: str, out_vcf: str):
```

Called as such

```
main(args.input_vcf, args.out_vcf)
```

Then typing and potential unit testing becomes easier